test/dotnet-cache.Tests/GivenDotnetCachesAndPublishesProjects.cs

```diff
@@ -88,8 +89,8 @@ public void AppFailsDueToMissingCache()
                 .CreateInstance()
                 .WithSourceFiles()
                 .Root.FullName;
-            var profileProject = Path.Combine(profileProjectPath, $"{profileProjectName}.xml");
-
+            var profileProject = Path.Combine(profileProjectPath, "NewtonsoftFilterProfile.xml");
+Console.WriteLine("File is {0}",profileProject);
```

Comment:
Not sure this is necessary.  If this run is green, let's merge it as is. But let's clean this up in your subsequent PR.